link_drag0n.js

```diff
@@ -0,0 +1,76 @@
+
+const fs = require('fs');
+let { Agent } = require('./agents/AddressableAgent.js');
+
+const subprocess = require('child_process');
+
+let agent = new Agent("LinkDragonAgent", "hi");
+agent.connect();
+agent.login();
+
+let allowedUsers = ['gene', 'liz', 'roan'];
```

Comment:
let's add a file that lists all allowed users so we can append names to the file

Reply:
or remove this requirement entirely, frankly